src/main/java/frc/robot/hardware/phoenix6/motor/TalonFXMotor.java

```diff
@@ -82,17 +82,6 @@ public void resetPosition(Rotation2d position) {
 		motor.setPosition(position.getRotations());
 	}
 
-
-	@Override
-	public void stop() {
-		motor.stopMotor();
-	}
-
-	@Override
-	public void setPower(double power) {
-		motor.set(power);
-	}
-
```

Comment:
This needs a replacement in the interface to ensure all motors support setting their power and stopping them